e3sm_to_cmip/util.py

```diff
@@ -109,20 +109,15 @@ def print_message(message, status="error"):
 # ------------------------------------------------------------------
 
 
-def setup_cmor(var_name, table_path, table_name, user_input_path):
+def setup_cmor(var_name, table_path, table_name, user_input_path, cmor_log_dir):
     """
-    Sets up cmor and logging for a single handler
-    """
-    var_name = str(var_name)
-    table_path = str(table_path)
-    table_name = str(table_name)
-    user_input_path = str(user_input_path)
-
-    logfile = os.path.join(os.getcwd(), "logs")
-    if not os.path.exists(logfile):
-        os.makedirs(logfile)
```

Comment:
Removed this section of code because the CMOR logs directory is already created in `E3SMtoCMIP._setup_dirs_with_paths()`.